src/style.css

```
@@ -0,0 +1,116 @@
+.home-page {
```

add the styles to html selector
```suggestion
html {
```